app/models/concerns/attribute_displayable.rb

```diff
@@ -0,0 +1,13 @@
+module AttributeDisplayable
```

Comment:
This is the only one whose name is a bit confusing to me. What else would go in the concern?